docs/website/docs/user-guides/advanced/connecting-a-service.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Connecting to a service
+sidebar_position: 1
+---
+
+This tutorial will show you how you can connect your Go application to a mongodb service.
+
+Building on top of the [Go quickstart guide](../quickstart/go.md), we will extend the application to check if it is connected to a mongodb service.
+
+### Pre-requisites
+1. Install Service Binding Operator on your cluster.  See [installation instruction](https://operatorhub.io/operator/service-binding-operator).
+
+2. Install Percona Server Mongodb operator on your cluster. See [installation instruction](https://operatorhub.io/operator/percona-server-mongodb-operator).
+:::note
+The operator will be installed in a new namespace called "my-percona-server-mongodb-operator" and will be usable from that namespace only.
+:::
+3. Create a mongodb service.
+
+import CreateMongodbService from './_create-mongodb-service.mdx';
+
+<CreateMongodbService/>
+
+### Implement the code logic
+:::note
+If you're already running `odo dev` in a terminal, exit it and start afresh.
```

Comment:
isn't this implied? no need for note i believe.

Reply:
At the beginning we mentioned we're building this on top of the quickstart guide. The last step there is to run `odo dev`. If it is still running, this note makes sense IMO.

Reply:
That makes sense. I reccomend we remove all informal language however like "you're", so I reccomend this change: